src/backend/tasks_io/handlers/frc_api.py

```diff
@@ -371,15 +372,17 @@ def event_details(event_key: EventKey) -> Response:
     if not Event.validate_key_name(event_key):
         return make_response(f"Bad event key: {Markup.escape(event_key)}", 400)
 
-    df = DatafeedFMSAPI(save_response=True)
+    fms_df = DatafeedFMSAPI(save_response=True)
+    nexus_df = DatafeedNexus()
```

Comment:
Is there a point in this given `src/backend/tasks_io/handlers/nexus_api.py` exists? alternatively only updating event teams here would avoid the race condition below.

Reply:
mmm, yeah, guess I probably should split this out into its out daily cronjob for all events